src/locales/en.json

```diff
@@ -50,6 +50,13 @@
         "synchronize": "Synchronize now"
       }
     },
+    "folder": {
+      "title": "Folder",
+      "link" : {
+        "default": "Open the %{name} folder",
```

Comment:
I don't think we need a different message. The folder is already created and is already accessible, even if the konnector is still fetching bills.